rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -752,6 +753,11 @@ public int rpcPort() {
                 configFromFiles.getInt(PROPERTY_RPC_PORT) : DEFAULT_RPC_PORT;
     }
 
+    public List<String> rpcHost() {
```

Comment:
maybe plural? e.g. `rpcHosts`.

Reply:
It's used to validate the Host header, I would keep it on singular